src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/AndroidLinkContext.cs

```diff
@@ -12,5 +13,6 @@ public AndroidLinkContext (Pipeline pipeline, AssemblyResolver resolver, ReaderP
 			: base (pipeline, resolver, readerParameters, factory) {}
 
 		public bool PreserveJniMarshalMethods { get; set; }
+
```

Comment:
I think there are maybe some leftover changes in this file, you could remove.

Reply:
I had to fix it :)